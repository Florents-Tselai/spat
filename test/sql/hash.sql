/* -------------------- HASHES -------------------- */

SELECT HSET('h1', 'f1', 'Hello');
SELECT HSET('h1', 'f2', 'World');

SELECT HGET('h1', 'f1'); -- Hello
SELECT HGET('h1', 'f2'); -- World
SELECT HGET('h1', 'f3'); -- NULL


SELECT HGET('h3', 'f1'); -- NULL
