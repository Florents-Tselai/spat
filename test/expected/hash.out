/* -------------------- HASHES -------------------- */
SELECT HSET('h1', 'f1', 'Hello');
 hset 
------
 
(1 row)

SELECT HSET('h1', 'f2', 'World');
 hset 
------
 
(1 row)

SELECT HGET('h1', 'f1'); -- Hello
 hget  
-------
 Hello
(1 row)

SELECT HGET('h1', 'f2'); -- World
 hget  
-------
 World
(1 row)

SELECT HGET('h1', 'f3'); -- NULL
 hget 
------
 
(1 row)

SELECT HGET('h3', 'f1'); -- NULL
 hget 
------
 
(1 row)

